libraries/AP_Math/tests/test_geodesic_grid.cpp

```diff
@@ -226,7 +226,7 @@ static TestParam hardcoded_vectors[] = {
     /* 3 * m_a + 2 * m_b 0 * m_c for T_4 */
     {{M_GOLDEN, -4.0f * M_GOLDEN -1.0f, 1.0f}, -1, {16, 18, -1}},
     /* 2 * m_c + (1 / 3) * m_b + .1 * c for T_13 */
-    {{-.2667f, .1667f * M_GOLDEN, 2.2667f * M_GOLDEN + .1667f}, 55},
+    {{-.2667f, 0.1667f * M_GOLDEN, 2.2667f * M_GOLDEN + 0.1667f}, 55},
```

Comment:
I think the first item here was missed.

Reply:
@rmackay9 san
Thank you for the review.
I have incorporated your comments.